### DISCUSSION TITLE
Sari Sari Channel

### DISCUSSION THREAD

**Discussion by thedarkdragon11** [iptv-org/community#671] [Q&A]
**Channel Name:** Sari-sari Channel
**Channel ID:** SariSariChannel.ph
**Country:** Philippines
**Language:** Tagalog
**External Link (optional):** xxx
**Notes (optional):** Classic shows channel in the Philippines.

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19669
